test/server.spec.ts

```diff
@@ -60,3 +94,92 @@ describe('GET /readiness', () => {
     expect(res.text).toEqual('OK')
   })
 })
+
+describe('POST /bzz', () => {
+  it('should store and retrieve actual directory with index document', async () => {
+    const batch = getPostageBatch()
+
+    const path = 'test/data/'
+    const dir = `./${path}`
+    const fileName = '1.txt'
+    const directoryStructure = await makeCollectionFromFS(dir)
+
+    const { reference } = await beeProxy.uploadCollection(batch, directoryStructure, {
+      indexDocument: `${fileName}`,
+    })
+
+    const file1 = await bee.downloadFile(reference)
+    expect(file1.name).toEqual(fileName)
+    expect(file1.data.text()).toMatch(/^1abcd\n$/)
+  })
+})
+
+describe('GET /bytes/:reference/', () => {
+  it('should retrieve bytes from the proxy', async () => {
+    const batch = getPostageBatch()
+    const path = 'test/data/'
+    const dir = `./${path}`
+    const fileName = '1.txt'
+    const directoryStructure = await makeCollectionFromFS(dir)
```

Comment:
You are using these 5 lines of code in almost every test. I would suggest you to create a helper function that return the `directoryStructure`